src/models/generated/AC1A.jl

```diff
@@ -0,0 +1,225 @@
+#=
+This file is auto-generated. Do not edit.
+=#
+"""
+    mutable struct AC1A <: AVR
+        Tr::Float64
+        Tb::Float64
+        Tc::Float64
+        Ka::Float64
+        Ta::Float64
+        Va_lim::Tuple{Float64, Float64}
+        Te::Float64
+        Kf::Float64
+        Tf::Float64
+        Kc::Float64
+        Kd::Float64
+        Ke::Float64
+        E_sat::Tuple{Float64, Float64}
+        Se::Tuple{Float64, Float64}
+        Vr_lim::Tuple{Float64, Float64}
+        V_ref::Float64
+        saturation_coeffs::Tuple{Float64, Float64}
+        ext::Dict{String, Any}
+        states::Vector{Symbol}
+        n_states::Int64
+        internal::InfrastructureSystemsInternal
+    end
+
+This excitation systems consists of an alternator main exciter feeding its output via non-controlled rectifiers.
+The exciter does not employ self-excitation, and the voltage regulator power is taken from a source that is not affected by external transients.
+Parameters of IEEE Std 421.5 Type AC1A Excitacion System. ESAC1A in PSSE and PSLF
+
+# Arguments
+- `Tr::Float64`: Regulator input filter time constant in s, validation range: (0, nothing)
+- `Tb::Float64`: Regulator denominator (lag) time constant in s, validation range: (0, nothing)
+- `Tc::Float64`: Regulator numerator (lead) time constant in s, validation range: (0, nothing)
+- `Ka::Float64`: Regulator output gain, validation range: (0, nothing)
+- `Ta::Float64`: Regulator output time constant in s, validation range: (0, nothing)
+- `Va_lim::Tuple{Float64, Float64}`: Limits for regulator output (Va_min, Va_max)
+- `Te::Float64`: Exciter field time constant in s, validation range: (&quot;eps()&quot;, nothing)
+- `Kf::Float64`: Rate feedback excitation system stabilizer gain, validation range: (0, nothing)
+- `Tf::Float64`: Rate feedback time constant, validation range: (&quot;eps()&quot;, nothing)
```

Comment:
@jd-lara @daniel-thom 
Any idea on how to fix this comment to not output `&quot`. In this case the validation range has to be strictly greater than zero (i.e. `eps()`).

Reply:
This is a problem in the IS Mustache template definition.  The `valid_range` and `validation_action` variables need to be enclosed in three curly braces instead of two.

Current:
```
- `{{name}}::{{{data_type}}}`{{#comment}}: {{{comment}}}{{/comment}}{{#valid_range}}, validation range: {{valid_range}}{{/valid_range}}{{#validation_action}}, action if invalid: {{validation_action}}{{/validation_action}}
```
Fix:
```- `{{name}}::{{{data_type}}}`{{#comment}}: {{{comment}}}{{/comment}}{{#valid_range}}, validation range: {{{valid_range}}}{{/valid_range}}{{#validation_action}}, action if invalid: {{{validation_action}}}{{/validation_action}}
```

Please test this locally and then we can fix push a fix in IS.

Reply:
This work locally and create a documentation of the form:
`- `R::Float64`: Speed droop parameter, validation range: ("eps()", nothing)`

I will not do the change now since it is failing the `test_generate_structs.jl` that it seems to compare the docstring, and will probably fail the test here in Travis.

Reply:
Unfortunately, we will have to make the change in IS, create a new tag, update to that tag in PSY, regenerate the structs, commit that change..